src/Geometry/Transform.jl

```diff
@@ -233,6 +234,15 @@ function Transform(rotation::AbstractArray{T,2}, translation::AbstractArray{T,1}
         translation[1], translation[2], translation[3], one(T))
 end
 
+
+# define some utility functions 
+right(t::Transform{<:Real}) = normalize(Vec3(t[1,1], t[2,1], t[3,1]))
```

Comment:
documentation strings would be helpful here to explain what these functions do and what they are used for. Graphics people will know but optics people might not.